compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUEnums.td

```diff
@@ -123,7 +127,9 @@ def IREEGPU_MMAIntrinsic : IREEGPU_I32MmaEnumAttr<"MMAIntrinsic",
       MFMA_I32_32x32x16_I8,
       WMMA_F32_16x16x16_F16,
       WMMA_F16_16x16x16_F16,
-      WMMA_I32_16x16x16_I8
+      WMMA_I32_16x16x16_I8,
+      MFMA_I32_16x16x16_I8,
+      MFMA_I32_32x32x8_I8
```

Comment:
nit: Can we put the MFMAs all next to each other, even though they are for different architectures